src/molgenis_fdp_harvester/ckan_harvest/molgenis_dcat_profile.py

```diff
@@ -0,0 +1,113 @@
+# SPDX-FileCopyrightText: Open Knowlege
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# This material is copyright (c) Open Knowledge.
+# It is open and licensed under the GNU Affero General Public License (AGPL) v3.0
+# Original location of file: https://raw.githubusercontent.com/ckan/ckanext-dcat/master/ckanext/dcat/profiles/euro_dcat_ap.py
+#
+# Modified by Stichting Health-RI to remove dependencies on CKAN
+
+import json
+from typing import Dict
+
+from rdflib import term, URIRef, BNode, Literal
+
+import logging
+
+
+# import ckantoolkit as toolkit
+
+# from ckan.lib.munge import munge_tag
+
+# from ckanext.dcat.utils import (
+#     resource_uri,
+#     DCAT_EXPOSE_SUBCATALOGS,
+#     DCAT_CLEAN_TAGS,
+#     publisher_uri_organization_fallback,
+# )
+from .baseparser import RDFProfile, URIRefOrLiteral, CleanedURIRef, munge_tag
+from .baseparser import (
+    RDF,
+    XSD,
+    SKOS,
+    RDFS,
+    DCAT,
+    DCT,
+    ADMS,
+    XSD,
+    VCARD,
+    FOAF,
+    SCHEMA,
+    SKOS,
+    LOCN,
+    GSP,
+    OWL,
+    SPDX,
+    GEOJSON_IMT,
+    namespaces,
+)
+
+log = logging.getLogger(__name__)
+
+# config = toolkit.config
+DCAT_CLEAN_TAGS = False
+NORMALIZE_CKAN_FORMAT = True
+
+
+DISTRIBUTION_LICENSE_FALLBACK_CONFIG = "ckanext.dcat.resource.inherit.license"
+
+
+class MolgenisEUCAIMDCATAPProfile(RDFProfile):
+    """
+    An RDF profile based on the DCAT-AP for data portals in Europe
+
+    More information and specification:
+
+    https://joinup.ec.europa.eu/asset/dcat_application_profile
+
+    """
+
+    def parse_dataset(self, dataset_dict: Dict, dataset_ref: URIRef):
+        # dataset_dict["extras"] = []
+        # dataset_dict["resources"] = []
+        dataset_dict["uri"] = str(dataset_ref)
+        # Basic fields
+        for key, predicate in (
+            ("name", DCT.title),
+            ("description", DCT.description),
+        ):
+            value = self._object_value(dataset_ref, predicate)
+            if value:
+                dataset_dict[key] = value
+
+        # TODO replace tags by keywords
+
+        # Tags
+        # replace munge_tag to noop if there's no need to clean tags
+        # do_clean = toolkit.asbool(config.get(DCAT_CLEAN_TAGS, False))
+        do_clean = DCAT_CLEAN_TAGS
+        tags_val = [
+            munge_tag(tag) if do_clean else tag for tag in self._keywords(dataset_ref)
+        ]
+        tags = [{"name": tag} for tag in tags_val]
+        # dataset_dict["tags"] = tags
+
+        # These values are fake. They need to be made "real"
+        log.warn("Filling in fake values")
```

Comment:
Want to make this values not configurable?

Reply:
They should be replaced by the proper rdf terms once they are finalized. Leaving it like this for now but agree should be made more dynamic in the future.